index.html

```diff
@@ -4,37 +4,42 @@
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bootstrap-icons@1.9.1/font/bootstrap-icons.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="styles.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
     <title>Portafolio</title>
 </head>
-
 <body>
     <header>
      <section class="home">
+        <div class="nav-bar">
+            <a href="https://github.com/cvalencia1991">Cesar Valencia</a>
+          <div>
+           <!--menu--> <i class="bi bi-list menu"></i>
+          </div>
+    </div>
+    </section>
+    </header>
     <div class="cover">
-        <div class="bar-status">
-            <time> 9:41</time>
-            <img src="statusphone.jpg" alt="statusphone">
-        </div>
-        <div class="main">
-            <p>cesar valencia</p>
-            <button>hamburger</button>
-        </div>
         <div class="content">
-            <h1>Hey there, i´m cesar</h1>
-            <h2> im a software engineer and mecatronic ensgineer</h2>
-            <p><i>can help you build a product , 
-                feature or website Look through some of my work and experience! 
-                If you like what you see and have a project you need coded, 
-                don’t hestiate to contact me.</i></p>
+            <h1>Hey there,<br>I´m cesar</h1>
+            <h2> I'am a software engineer and mecatronic engineer</h2>
+            <p><br><em>can help you build a product,feature or website Look
+                through some of my work and experience!If you like what you
+                see and have a project you need coded, don’t hestiate to
+                contact me.</em></p>
         </div>
-        <div>
-            <input type="image"  alt="githublink" src="imgs/github image.jpg"/>
-            <input type="image"  alt="linkeinlink" src="imgs/inlinked image.jpg"/>
-            <input type="image"  alt="twitterlink" src="imgs/twitter img.jpg"/>
-        </div>
-    </div> 
-     </section>
-    </header>
+    <h3>Let´s Connect</h3>
+    <div>
+          <img src="imgs/githubimg.png" alt="githubimage">
+          <img  src="imgs/linkeinimg.png" alt="linkedinimage">
+          <img src="imgs/twitter.png" alt="twitterimg">
+    </div>
+</div>
```

Comment:
- [ **OPTIONAL** ] **RECOMMENDED:** Kindly make sure all section container is wrapped in a `<section>` tag as required for the project and for good semantics tags. You can just change this ` <div class="cover">` to a `<section class="cover"> .... </section>` tag as this change was missed by the previous reviewer